keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -234,21 +234,63 @@ def test_saved_model(self, save_format, filename):
 
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
-    def test_backbone_preset_weight_loading(self):
-        # Check that backbone preset weights loaded correctly
-        # TODO(lukewood): need to forward pass test once proper weights are
-        # implemented
-        keras_cv.models.RetinaNet.from_preset(
-            "resnet50_v2_imagenet",
-            num_classes=20,
-            bounding_box_format="xywh",
-        )
+    def test_backbone_preset(self):
+        weights = [
+            "csp_darknet_tiny",
+            "efficientnetv2_s",
+            "efficientnetv2_b0",
+            "mobilenet_v3_small",
+            "resnet50",
+            "resnet50_v2",
+        ]
+        for weight in weights:
+            model = keras_cv.models.RetinaNet.from_preset(
+                weight,
+                num_classes=20,
+                bounding_box_format="xywh",
+            )
+            xs, _ = _create_bounding_box_dataset(bounding_box_format="xywh")
+            output = model(xs)
+            self.assertEqual(output["box"].shape, (xs.shape[0], 49104, 4))
```

Comment:
What are these magic constants?

Reply:
4 is the number of parameters in a box, 49104 is the number of anchors for a 512x512 image.
(Probably we should either comment this or make these named constants)

Reply:
I will add comments in the follow-up PR!